src/TodoList.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+let todoList = [
+  {
+    id: "1",
+    title: "Complete React assignment",
+  },
+  { id: "2", title: "Pick up lunch" },
+  {
+    id: "3",
+    title: "Play Zelda",
+  },
+];
+
+function TodoList() {
+  return (
+    <ul>
+      {todoList.map(function (toDoItem) {
+        return <li key={toDoItem.id}>{toDoItem.title}</li>;
+      })}
```

Comment:
we can use arrow function in map

```
 {todoList.map((todo) => (
        <li key={todo.id}>{todo.title}</li>
      ))}
```

some examples can be find here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map